README.md

```diff
@@ -52,20 +52,20 @@ dispatch:
 ```
 
 ## Step 2: Installing the Package
-Include the following fivetran_log package version in your `packages.yml`
+Include the following fivetran package version in your `packages.yml`
 > Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions, or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
 ```yaml
 packages:
-  - package: fivetran/fivetran_log
-    version: [">=0.7.0", "<0.8.0"]
+  - package: fivetran/fivetran
```

Comment:
This will still be `fivetran_log` since the dbt hub name is tied to the repo name. I checked with Shiva and this is okay to keep the naming as log here.